src/components/infoCard/InfoCard.tsx

```diff
@@ -4,7 +4,9 @@ import { StyleSheet, View } from 'react-native';
 import { Icon as RNEIcon } from 'react-native-elements';
 
 import { colors, consts, normalize } from '../../config';
-import { Address, Contact, OpeningHour, WebUrl } from '../../types';
+import { mergeWebUrls } from '../../helpers/mergeWebUrls';
+import { Address, OpeningHour } from '../../types';
+import { WebUrlProps } from '../../types/WebUrlProps';
```

Comment:
what is the difference between them to be imported differently?

Reply:
This is because I didn't add `WebUrlProps` to index.js. But now I've moved `mergeWebUrls` to `linkHelper.js` and added `WebUrlProps` back to `InfoCard.tsx` since I only use it in one place. 

https://github.com/smart-village-solutions/smart-village-app-app/commit/0de11ba779b6755711897c4ca4939cada2092108